src/pages/ContentScripts/features/developer-hovercard-info/index.tsx

```diff
@@ -31,48 +42,93 @@ const renderTo = (container: HTMLElement, developerName: string, openrank: strin
   ReactDOM.render(<View developerName={developerName} openrank={openrank} />, openRankContainer);
 };
 
-const init = async (): Promise<void> => {
+const elementReadyWithTimeout = async (selector: string, options: { stopOnDomReady: boolean }, timeout: number) => {
+  return Promise.race([
+    elementReady(selector, options),
+    new Promise((_, reject) => setTimeout(() => reject(new Error(`Timeout waiting for ${selector}`)), timeout)),
+  ]);
+};
+
+const processElement = (element: Element) => {
+  const hovercardUrl = element.getAttribute('data-hovercard-url');
+  if (!hovercardUrl || !hovercardUrl.startsWith('/users')) {
+    return;
+  }
+
   let abortController = new AbortController();
-  const hovercardSelector = '[data-hovercard-type="user"]';
-  document.querySelectorAll(hovercardSelector).forEach((element) => {
-    // isProcessing is used to Prevent OpenRank from adding duplicates
-    element.addEventListener('mouseover', async () => {
-      abortController.abort();
-      abortController = new AbortController();
-      const signal = abortController.signal;
-
-      const developerName = getDeveloperName(element as HTMLElement) as string;
-
-      // Create a unique identifier for the popover
-      const popoverId = `popover-${developerName}`;
-
-      // Get the floating card container
-      const $popoverContainer =
-        'body > div.logged-in.env-production.page-responsive > div.Popover.js-hovercard-content.position-absolute > div > div > div';
-      const popover = await elementReady($popoverContainer, { stopOnDomReady: false });
-
-      const openRankDiv = popover?.querySelector('.hypercrx-openrank-info');
-      const existingDeveloperName = openRankDiv?.getAttribute('data-developer-name');
-      if (existingDeveloperName === developerName) {
-        return;
-      }
-      openRankDiv?.remove();
 
-      // Set the popover's unique identifier
-      // make the current OpenRank information and person match
-      popover?.setAttribute('data-popover-id', popoverId);
+  element.addEventListener('mouseover', async () => {
+    abortController.abort();
+    abortController = new AbortController();
+    const signal = abortController.signal;
 
```

Comment:
` await new Promise(resolve => setTimeout(resolve, 600));`
The current situation is that if the mouse hovers over the avatar to the name or over the name to the avatar, there will be two instances of openrank. My idea is to add this code to ensure that the mouse hovers over the corresponding position instead of quickly skimming, and then proceed with the following operations. You can give it a try.